core/src/main/scala/io/snappydata/TableStatsProviderService.scala

```diff
@@ -193,12 +193,14 @@ trait TableStatsProviderService extends Logging {
     if (!doRun) return (Map.empty, Map.empty, Map.empty)
     // val samples = getSampleTableList(snc)
     tableStats.foreach { stat =>
-      aggregatedStats.get(stat.getTableName) match {
-        case Some(oldRecord) =>
-          aggregatedStats.put(stat.getTableName, oldRecord.getCombinedStats(stat))
-        case None =>
-          aggregatedStats.put(stat.getTableName, stat)
-      }
+        if (!stat.getTableName.contains("SNAPPYSYS_INTERNAL____SINK_STATE_TABLE")) {
```

Comment:
@snappy-sachin for any other internal tables, can change this to be: if (!stat.getTableName.contains("SNAPPYSYS_INTERNAL")